frontend/app/dashboard/src/views/dashboard/settings/modules/members/RecordsSettingsView.vue

```diff
@@ -116,7 +116,7 @@
             <h2>Gezondheid, hygiëne &amp; slapen</h2>
 
             <Checkbox :checked="getBooleanType(RecordType.CovidHighRisk)" @change="setBooleanType(RecordType.CovidHighRisk, $event)">
-                Hoge riscogroep voor coronavirus
+                Hoge risicogroep voor coronavirus
```

Comment:
De kans is groot dat deze typfout ook nog ergens anders staat. Kan nu niet checken (op reis), maar als je in heel het project zoekt, zou je het moeten vinden :)

Reply:
Ik heb het niet op andere plaatsen teruggevonden. Maar daar staat tegenover dat ik (nog) geen held ben in Github en zelfs de zoekfunctie gebruiken al een hele opdracht vind. :)